gridy/Sources/Presentations/PlanBoard/ShareImageView.swift

```diff
@@ -2,32 +2,259 @@
+                        )
+                }
+                .buttonStyle(.link)
+                .onHover { hover in
+                    shareHovered = hover
+                }
+                Spacer()
+                Button(action: {
                     dismiss()
+                }) {
+                    RoundedRectangle(cornerRadius: 5)
+                        .foregroundStyle(cancelHovered ? Color.itemHovered : Color.item)
+                        .frame(width: 80, height: 24)
+                        .overlay(
+                            Text("Cancel")
+                                .font(.system(size: 13))
+                                .foregroundStyle(Color.button)
+                        )
+                }
+                .buttonStyle(.link)
+                .onHover { hover in
+                    cancelHovered = hover
+                }
+                Button(action: {
+                    saveImage()
+                }) {
+                    RoundedRectangle(cornerRadius: 5)
+                        .foregroundStyle(createHovered ? Color.itemHovered : Color.item)
+                        .frame(width: 80, height: 24)
+                        .overlay(
+                            Text("Save")
+                                .font(.system(size: 13))
+                                .foregroundStyle(Color.button)
+                        )
+                }
+                .buttonStyle(.link)
+                .onHover { hover in
+                    createHovered = hover
                 }
             }
-            ToolbarItem(placement: .confirmationAction) {
-                Button("Continue") {
-                    dismiss()
+            .padding(.horizontal, 16)
+        }
+        .frame(width: 468, height: 523)
+        .padding(16)
+        .background(Color.gray.opacity(0.3))
+    }
+    
+    /// Appkit을 이용한 share image
+    @MainActor func shareImage() {
+        if let contentView = NSApp.keyWindow?.contentView {
+            let image = ImageRenderer(content: TimelineLayoutView().environment(\.colorScheme, colorScheme)).nsImage
+            let sharingServicePicker = NSSharingServicePicker(items: [image!])
+            
+            sharingServicePicker.show(relativeTo: .zero, of: contentView, preferredEdge: .minY)
+        }
+    }
+    
+    func formattedDate(date: Date) -> String {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy.MM.dd."
+        return formatter.string(from: date)
+    }
+    
+    @MainActor func saveImage() {
+        let savePanel = NSSavePanel()
+        savePanel.allowedContentTypes = [.png]
+        savePanel.canCreateDirectories = true
+        savePanel.isExtensionHidden = false
+        savePanel.title = "Save your image"
+        savePanel.message = "Choose a folder and a name to store the image."
+        savePanel.begin { response in
+            if response == .OK, let url = savePanel.url {
```

Comment:
조건문에 여러 조건이 들어갈 경우에는 콤마마다 개행하는 것이 어떤가요!
```suggestion
            if response == .OK, 
               let url = savePanel.url {
```